apps/web/src/translations/vendors/dayjs.ts

```diff
@@ -1,17 +1,11 @@
 export const dayjsLocales: Record<string, () => Promise<any>> = {
     en: () => import('dayjs/locale/en'),
     ko: () => import('dayjs/locale/ko'),
-    jp: () => import('dayjs/locale/ja'),
+    ja: () => import('dayjs/locale/ja'),
 };
 
 const cachedDayjsLocales: Record<string, boolean> = {};
 
-export const dayjsLocaleMap = {
-    en: 'en',
-    ko: 'ko',
-    jp: 'ja',
-};
```

Comment:
description: Removed unnecessary object. It was generated to reduce confusion which from different code names, jp and ja.